### PR TITLE
Handle (un)signed extensions of integer types.

### PR DESCRIPTION
**Pull request by tj90241**

### CODE REVIEW

File: test/intcast.c [hunk 1]
```diff
@@ -0,0 +1,26 @@
+// Copyright 2012 Rui Ueyama <rui314@gmail.com>
+// This program is free software licensed under the MIT license.
+
```
Comment (andrewchambers) [inline test/intcast.c]: Adding a new file here is not needed, there is already a test/cast.c

**Comment by tj90241** [comment]
Good point; moved the TCs to test/cast.c per your advice.

File: test/cast.c [hunk 2]
```diff
@@ -12,4 +26,7 @@ void testmain(void) {
     int a[3];
     *(int *)(a + 2) = 5;
     expect(5, a[2]);
+
```
Comment (andrewchambers) [inline test/cast.c]: The other test files don't really have white spaces here. I'd wait for a comment from Rui about the white spaces.

Sorry about being picky - Consistency keeps things highly readable.

**Comment by andrewchambers** [comment]
Sorry, I was commenting on the wrong diff, then on the wrong file. I'm not 100 percent sure about the whitespaces in the tests, hopefully its consistent with other files. 

(probably spammed your inbox.)

**Comment by rui314** [comment]
Thanks. Squashed them and committed as fa21c593d9ebabc5de3ee1b70845050b3c5bcc99.

**Pull request closed without merging**